addons/package_hierarchy/models/stock_quant_package.py

```diff
@@ -38,46 +37,47 @@ class QuantPackage(models.Model):
             "and within the contained package hierarchy."
         ),
     )
-    children_ids = fields.One2many("stock.quant.package", "parent_id", "Contained Packages")
-    depth = fields.Integer(compute="_compute_depth", store=True)
+    # Renaming to child_ids as from hierarchy tree view should have child_ids for the new tree view
```

Comment:
This view is added by a closed source module, we shouldn't be referencing it here.

As said in the other PR, we should discuss this more, but renaming our fields to work around a view doesn't feel right.

If we go by what the story is for then the field should be `x_children_ids` (unless `child_ids` is a reserved field name in Odoo, I can't remember)

Reply:
Removed the comment :)